### PR TITLE
[CH-264] support more json functions

### PR DESCRIPTION
**Pull request by lgbo-ustc**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)
Support more json functions

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
unit tests

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

test by CH[[264]]

### CODE REVIEW

**Comment by lgbo-ustc** [comment]
backend update https://github.com/Kyligence/ClickHouse/pull/264

File: gluten-ut/spark32/src/test/scala/org/apache/spark/sql/GlutenJsonFunctionsSuite.scala [hunk 1]
```diff
@@ -14,8 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.apache.spark.sql
 
+import org.apache.spark.sql.JsonFunctionsSuite
+
```
Comment (zzcclp) [inline gluten-ut/spark32/src/test/scala/org/apache/spark/sql/GlutenJsonFunctionsSuite.scala:20]: remove

File: gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala [hunk 2]
```diff
@@ -143,6 +143,8 @@ object ExpressionMappings {
   // JSON functions
   final val GET_JSON_OBJECT = "get_json_object"
   final val JSON_ARRAY_LENGTH = "json_array_length"
+  final val TO_JSON = "to_json"
```
Comment (zzcclp) [inline gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala:146]: @lviiii please check whether velox backend supports these function.

Comment (lviiii) [reply gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala:146]: Velox backend don't support these function and Gluten will take the fallback strategy automatically.

**Review by zzcclp — APPROVED** [review]
LGTM

**Pull request merged**